### PR TITLE
Iae 34767/icons

### PR DESCRIPTION
**Pull request by ascreven-gsa**
Refactor icons: https://cm-jira.usa.gov/browse/IAE-34767

- create template which shows all icons variants but can be pulled in throughout Sam Styles and update documentation accordingly
- update all instances of sds, fa, and fas icons in Sam Styles to call in Icon Component

Checked in IE, Edge, Chrome, Safari, and Firefox

### CODE REVIEW

File: src/components/02-structure/04-seal/seal.njk [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="height-mobile bg-accent-cool-light padding-y-4">
 
   <div class="sds-seal">
-    <i class="sds fa-flag fa-2x margin-y-1 text-secondary"></i>
```
Comment (AlejandroColinaGSA) [inline src/components/02-structure/04-seal/seal.njk:4]: original utility classes (margin-y-1 text-secondary) missing

File: src/components/03-components/01-actions/_actions--single.njk [hunk 2]
```diff
@@ -1,3 +1,3 @@
  <button class="sds-button sds-button--circular sds-button--{{style}}" aria-label="{{label}}">
-    <i class="sds fa-ellipsis fa-fw" data-fa-transform="grow-5"></i>
+  {% render '@icon', icon %}
```
Comment (AlejandroColinaGSA) [inline src/components/03-components/01-actions/_actions--single.njk:2]: data-fa-transform="grow-5" missing

File: src/components/03-components/02-alerts/alerts--block.njk [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <h3>SDS Block Alert</h3>
 <div class="sds-alert--block">
   <div class="sds-alert--block-icon grid-col-auto">
-    <i class="sds fa-exclamation fa-2x"></i>
```
Comment (AlejandroColinaGSA) [inline src/components/03-components/02-alerts/alerts--block.njk:4]: I noticed fa-2x missing throughout all instances including other files, not sure if this is by design

File: src/components/03-components/02-alerts/alerts.njk [hunk 4]
```diff
@@ -1,21 +1,21 @@
 <h3>SDS Alert</h3>
 <div class="sds-alert">
   <div class="sds-alert-icon">
-    <i class="fa fa-exclamation"></i>
```
Comment (AlejandroColinaGSA) [inline src/components/03-components/02-alerts/alerts.njk:4]: missinf fa-excalmation

File: src/components/02-structure/04-seal/seal.njk (hunk 1 above)
Comment (ascreven-gsa) [reply src/components/02-structure/04-seal/seal.njk:4]: This are in the seal.config file

File: src/components/03-components/01-actions/_actions--single.njk (hunk 2 above)
Comment (ascreven-gsa) [reply src/components/03-components/01-actions/_actions--single.njk:2]: This is in the actions.config file

File: src/components/03-components/02-alerts/alerts--block.njk (hunk 3 above)
Comment (ascreven-gsa) [reply src/components/03-components/02-alerts/alerts--block.njk:4]: This is in alerts.config

File: src/components/03-components/02-alerts/alerts.njk (hunk 4 above)
Comment (ascreven-gsa) [reply src/components/03-components/02-alerts/alerts.njk:4]: also in alerts.config

**Pull request merged**